frontend_build/src/prettier-frontend.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-console */
```

Comment:
Shouldn't we be using logging.js anyways @rtibbles?

Reply:
All the local logging.js does is add colours, and prefix everything with `Kolibri: `, so I didn't think it was necessary here :)